sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DocumentAnalysisLiveTestBase.cs

```diff
@@ -100,8 +100,9 @@ protected DocumentModelAdministrationClient CreateDocumentModelAdministrationCli
         /// </summary>
         /// <param name="modelId">Model Id.</param>
         /// <param name="containerType">Type of container to use to execute training.</param>
+        /// <param name="buildMode">The technique to use to build the model. Defaults to <see cref="DocumentBuildMode.Template"/>.</param>
         /// <returns>A <see cref="DisposableBuildModel"/> instance from which the trained model ID can be obtained.</returns>
-        protected async Task<DisposableBuildModel> CreateDisposableBuildModelAsync(string modelId, ContainerType containerType = default)
+        protected async Task<DisposableBuildModel> CreateDisposableBuildModelAsync(string modelId, ContainerType containerType = default, DocumentBuildMode buildMode = default)
```

Comment:
This is an internal test method we use to make things simpler. Just updating it to default to `Template` when no build mode is specified.